protocol/triple/triple_protocol/codec.go

```diff
@@ -337,3 +338,193 @@ func errNotProto(message interface{}) error {
 	}
 	return fmt.Errorf("%T doesn't implement proto.Message", message)
 }
+
+// Definitions from dubbogo/grpc-go
+func GetArgType(v interface{}) string {
```

Comment:
For now, this function does not need to be exposed. We would expose it if needed in the future.